src/sbvr-api/permissions.ts

```diff
@@ -1256,6 +1256,16 @@ const $getApiKeyPermissions = (() => {
 											},
 										},
 									},
+									$or: [
```

Comment:
@thgreasi @Page- I am not sure about this specific addition, should we be doing a expiry check while doing a permissions query?

Reply:
I think that makes sense @nitishagar but you probably need to add the same check in the `is_of__role` part of the query as well.
And on top of that you can also add it to the `getApiKeyActorId()`, so that pine treats the api key as unknown in all cases.